### PR TITLE
add some maps in readme

### PR DESCRIPTION
**Pull request by wsdjeg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,15 +93,29 @@ Add this to your `.vimrc` file:
 
 `autocmd FileType java set omnifunc=javacomplete#Complete`
 
+To enable import R.java for an android project
```
Comment (brcolow) [inline README.md]: You can enable importing R.java for an Android project by using the maven-android-plugin by adding the following to your vimrc:

Comment (mattn) [inline README.md]: @brcolow Thanks to your correcting.

**Comment by wsdjeg** [comment]
map for remove is f6 not f5

**Comment by brcolow** [comment]
It looks like it was merged without my grammar corrections.

File: README.md (hunk 1 above)
Comment (brcolow) [inline README.md]: @mattn No problem - it's the least I can do :)

**Pull request merged**